docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/meta-service-administration/meta-metrics.md

```diff
@@ -0,0 +1,60 @@
+# Meta Metrics
+
+## Introduction
+
+Currently, **Nebula Graph** supports obtaining the basic performance metrics for the meta service via HTTP.
+
+Each performance metrics consists of three parts, namely `<counter_name>.<statistic_type>.<time_range>`.
+
+### Counter Names
+
+Each counter name is composed of the interface name and the counter name. Meta service only counts the heart beat. Currently, the supported interfaces are:
+
+```text
+meta_heartbeat_qps
+meta_heartbeat_error_qps
+meta_heartbeat_latency
+```
+
+### Statistics Type
+
+Currently supported types are SUM, COUNT, AVG, RATE and P quantiles (P99, P999, ..., P999999). Among which:
+
+- metrics have suffixes `_qps` and `_error_qps` support SUM, COUNT, AVG, RATE but don't support P quantiles.
+- metrics have suffixes `_latency` support SUM, COUNT, AVG, RATE, and P quantiles.
+
+### Time Range
+
+Currently supported time ranges are 60, 600, and 3600, which mean the last minute, the last ten minutes, and the last hour till now.
```

Comment:
Currently, the supported time ranges are 60s, 600s, and 3600s, which correspond to the last minute, the last ten minutes, and the last hour till now.